packages/apps/human-app/frontend/src/pages/worker/jobs/components/my-jobs/desktop/my-jobs-table.tsx

```diff
@@ -38,11 +38,11 @@ import { EscrowAddressSearchForm } from '@/pages/worker/jobs/components/escrow-a
 import { useRefreshTasksMutation } from '@/api/services/worker/refresh-tasks';
 
 const getColumnsDefinition = ({
-  resignJob,
+  oracle_address,
   refreshData,
   isRefreshTasksPending,
 }: {
-  resignJob: (assignment_id: string) => void;
+  oracle_address: string | undefined;
```

Comment:
You already pass a default empty string below
```suggestion
  oracle_address: string;
```